test/files/testmatrix_variationId.csv

```diff
@@ -0,0 +1,8 @@
+Identifier;Email;Country;Custom1;boolean;decimal;text;whole
```

Comment:
Usually we add those mocked file infos in a folder named `fixtures` which gives the context that those will be used for testing purposes. It seems better to me than `files`, which doesn't give many context. wdyt?

Reply:
👍